src/Excel.php

```diff
@@ -124,6 +125,23 @@ public function queue($export, string $filePath, string $disk = null, string $wr
         );
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function storeQueued($export, string $filePath, string $disk = null, string $writerType = null, $diskOptions = [])
```

Comment:
Sure. Any intensions/suggestions for the naming and location?

Reply:
I tried some ways to move the logic to a dedicated class. But it feels more like duplicating the `Excel` class since I have to inject all the classes (`Filesystem`, `Writer`) and all export information.

Do you have an idea to keep this stuff internal?

Reply:
I can refactor it later, so don't worry about it :)

Reply:
I implemented a helper in https://github.com/Maatwebsite/Laravel-Excel/pull/2288/commits/116f722589cff0434f7079dff646d8345f975c89. Maybe the naming is too general.